.npmignore

```diff
@@ -11,6 +11,9 @@ pids
 *.seed
 *.pid.lock
 
+# non-compiled code
+src
```

Comment:
Please include this – there is no reason to ignore source code given it runs in `node` on the server untranspiled.